src/main/java/com/aws/greengrass/util/EncryptionUtils.java

```diff
@@ -149,4 +152,89 @@ private static byte[] join(byte[] byteArray1, byte[] byteArray2) {
         System.arraycopy(byteArray2, 0, bytes, byteArray1.length, byteArray2.length);
         return bytes;
     }
+
+    /**
+     * Converts given encoded object to a PEM string.
+     *
+     * @param encodedObject   encoded entity
+     * @param pemBoundaryType encoding boundary of pem
+     * @return a PEM string
+     * @throws IOException IOException
+     */
+    public static String encodeToPem(String pemBoundaryType, byte[] encodedObject) throws IOException {
```

Comment:
Should we convert `pemBoundaryType` to an enum?

Reply:
no, absolutely not, it can be anything